pennylane_lightning/core/src/simulators/lightning_gpu/utils/MPILinearAlg.hpp

```diff
@@ -79,11 +79,11 @@ inline void SparseMV_cuSparseMPI(
             color = 1;
             SparseMV_cuSparse<index_type, Precision, CFP_t>(
                 localCSRMatrix.getCsrOffsets().data(),
-                localCSRMatrix.getCsrOffsets().size(),
+                static_cast<int64_t>(localCSRMatrix.getCsrOffsets().size()),
```

Comment:
I assume again we always need this type here?

Reply:
See above.